src/it/resources/data/typicalDataBundle.json

```diff
@@ -48,6 +54,43 @@
       "name": "Typical Course 2",
       "institute": "TEAMMATES Test Institute 1",
       "timeZone": "Asia/Singapore"
+    },
+    "typicalCourse1": {
+      "createdAt": "2012-04-01T23:58:00Z",
+      "id": "idOfTypicalCourse1",
+      "name": "Typical Course 1",
+      "institute": "TEAMMATES Test Institute 1",
+      "timeZone": "Africa/Johannesburg"
+    },
+    "typicalCourse2": {
+      "createdAt": "2012-04-01T23:59:00Z",
+      "id": "idOfTypicalCourse2",
+      "name": "Typical Course 2",
+      "institute": "TEAMMATES Test Institute 1",
+      "timeZone": "UTC"
+    },
+    "typicalCourse3": {
+      "createdAt": "2012-04-02T23:58:00Z",
```

Comment:
Can we use existing data for this?

If not possible, better to add a customised data bundle for your particular test. Adding extra data here is not free, this means extra time needed to persist this data for all tests that use the typical data bundle. This adds up quickly over time.

Reply:
Hey Samuel, thanks for your comments below, everything looks way better now..!

Anyway, for this, I think maybe using the customised data bundle is a better approach. I did try using the existing data but I feel like it's not very comprehensive.

Also, if we are going to add the customised data bundle, we need to update `getTypicalSqlDataBundle` in `BaseTestCase` class to work the same way as the existing data bundle.

What are your thoughts!

Reply:
Sure, but keep in mind that we also have unit tests now.